### ISSUE TITLE
There is not an option to exclude some css files to be injected

### ISSUE THREAD

**Issue by sebas0811buitrago** [marco-prontera/vite-plugin-css-injected-by-js#54]
Hi, I'm building a React library using this vite config:

```
import { defineConfig } from 'vite';
import { buildPlugin } from 'vite-plugin-build';
import react from '@vitejs/plugin-react';
import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';

const isWatch = process.env.npm_lifecycle_event === 'watch';

export default defineConfig({
  plugins: [
    react(),
    buildPlugin({
      fileBuild: {
        ignoreInputs: ['./src/playground/*', './src/stories/*'],
        emitDeclaration: true,
        esOutputDir: './build',
        commonJsOutputDir: false,
        ...(isWatch && { watch: {} })
      }
    }),
    cssInjectedByJsPlugin({})
  ]
});
```
My issue is that I would like to have an option in the `cssInjectedByJsPlugin`  to exclude some css files from being injected. Is there a way to do that?

**Comment by Marco-Prontera** [comment]
Hi, @sebas0811buitrago can you show me a screen of what it's inside your output directory?
How many CSS files without using `cssInjectedByJsPlugin ` do you have?

**Comment by johanelbg** [comment]
Hi! 

I'm currently in the exact same situation

I have two scss files:
-global.scss
-font.scss

global.scss and styles from components needs to be injected,
but font.scss shouldn’t, as I want to export it and use it afterwards

**Comment by marco-prontera** [comment]
@johanelbg new version published [v3.1.0](https://www.npmjs.com/package/vite-plugin-css-injected-by-js/v/3.1.0) that include a new option to allow this.
Hope this new option implemented helps you.

In the next few days I'll be very busy, so if there are any other problems, PRs are welcome.

**Comment by johanelbg** [comment]
Thank you, I ll have a look !
